autosklearn/pipeline/components/feature_preprocessing/no_preprocessing.py

```diff
@@ -8,10 +8,9 @@ class NoPreprocessing(AutoSklearnPreprocessingAlgorithm):
 
     def __init__(self, random_state):
         """ This preprocessors does not change the data """
-        self.preprocessor = None
 
     def fit(self, X, Y=None):
-        self.preprocessor = 0
+        self.preprocessor = 'passthrough'
```

Comment:
Hi Matthias, do you think you can provide more details about making this passthrough? I was not able to figure out the impact of this....

Reply:
It's somewhat more in line with sklearn's Pipeline and ColumnTransformer where 'passthrough' indicates that something should be passed through and not touched. This won't have any practical impact at all.